Task_DEV_4/Task_DEV_4/Rectangle.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+
+namespace Task_DEV_4
+{
+    class Rectangle
+    {
+        private double length;
+        private double width;
+
+        public Rectangle()
+        {
+
+        }
+
+        public Rectangle(double length, double width)
+        {
+            this.length = length;
+            this.width = width;
+        }
+
+        public void SettingFieldValues(double length, double width)
+        {
+            this.length = length;
+            this.width = width;
+        }
+
+        public double CalculateTheArea()
+        {
+            return length * width;
+        }
+
+        public double CalculateThePerimeter()
+        {
+            return (length + width) * 2;
+        }
+
+        public bool CheckForSquare()
+        {
+            bool isNotSquare = true;
```

Comment:
Better not to use inverse logic. Direct logic looks clearer. Instead of  `isNotSquare`, use `isSquare`.

Reply:
This method can look like:
```
public bool IsSquare()
{
    return length == width;
}
```

Reply:
Done.